### ISSUE TITLE
webview2 disabled mouse xy click API function ? 

### ISSUE THREAD

**Issue by haruuser** [MicrosoftEdge/WebView2Feedback#2261]
When deactivated, there is no way to click on mouse xy positioning in the deactivation tap other than the deactivation tap of the tap control.

In another Chrome-based browser eo.webview, there is a separate command to control the mouse on its own, but in webview2, there is no such command, so I don't know how to implement it.

In fact, when activating, you can process it with mousevent api, but what you want to process with that api is an activation mode in the visible space, so I don't think it's necessary. I think we need a clickable mode when deactivating.

Although there is a separate part called CoreWebView2MouseEventKindEnum, there is no sample code for this part and no idea how to use it.

**Comment by champnic** [comment]
Hey @haruuser - apologies, but I'm not sure I totally understand what you are asking for here. What do you mean by "deactivated" and "deactivation tap"? When you say "Mousevent api" is that a .NET event, or a javascript event?

**Comment by haruuser** [comment]
SendMouseEvent(MouseEventType.Down, New EO.Base.UI.MouseEventArgs(EO.Base.UI.MouseButtons.Left, 1, Pos.X, Pos.Y, 0))
Perhaps there is no such command inside. Do I have to process all mouse movements and click events in javascript?
ex) document.elementFromPoint(x, y).click();

**Comment by champnic** [comment]
@haruuser Yes, currently you'll have to use javascript to properly handle mouse and keyboard inside the WebView2.
